README.adoc

```diff
@@ -2,13 +2,43 @@
 
 This repository contains Dockerfiles and scripts to build base images for use within Stackable.
 
+== Prerequisites
+
+* Stackable Image Tools (`pip install image-tools-stackabletech`) https://github.com/stackabletech/image-tools
+* Docker including the `buildx` plugin: https://github.com/docker/buildx
+* Optional: https://github.com/redhat-openshift-ecosystem/openshift-preflight[OpenShift preflight tool] to verify an image for OpenShift
+
+== Build Product Images
+
+Product images are published to the `docker.stackable.tech` registry under the `stackable` organization by default.
+
+To build and push product images to the default repository, use the `build_product_images.py` like this:
+
+    bake --product zookeeper --image 0.0.0-dev --push
```

Comment:
Maybe it's just me, but this is a little confusing.

It says to use `build_product_images.py`, but goes on to use `bake`.

Reply:
I see this section was moved, so nothing technically _needs_ fixing in this PR.

Reply:
I am confused by this as well though. I'll ping @razvan - maybe he can help

Reply:
It's an omission. The py script doesn't exist anymore

Reply:
I removed the reference.

Reply:
f405a3c